internal/cmd/login.go

```diff
@@ -153,85 +156,57 @@ func login(cli *CLI, options loginCmdOptions) error {
 			}
 		}
 
-		loginReq.PasswordCredentials = &api.LoginRequestPasswordCredentials{
-			Name:     options.User,
-			Password: options.Password,
-		}
+		loginRes, err = lc.APIClient.Login(ctx, &api.LoginRequest{
+			PasswordCredentials: &api.LoginRequestPasswordCredentials{
+				Name:     options.User,
+				Password: options.Password,
+			},
+		})
+		if err != nil {
+			if api.ErrorStatusCode(err) == http.StatusUnauthorized || api.ErrorStatusCode(err) == http.StatusNotFound {
+				return &LoginError{Message: "your username or password may be invalid"}
```

Comment:
Not a blocker for merging this, but do you know what causes the `NotFound`? Seems like maybe we should be handling that as `StatusUnauthorized` so that someone can't detect when a user exists or not.

Reply:
This was moved from elsewhere, but `/api/login` should never return 404 for user discovery reasons and I don't think the endpoint ever does, so I'll remove that check.